asmcomp/polling.ml

```diff
@@ -0,0 +1,194 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*        Xavier Leroy and Damien Doligez, projet Cambium, INRIA Paris    *)
+(*                Damien Doligez, Inria                                   *)
```

Comment:
I don't think I need to be credited *twice* for code that was almost entirely written by Xavier...
```suggestion
```